tests/Unit/Fetcher/FeedFetcherTest.php

```diff
@@ -220,6 +220,17 @@ public function testNoFetchIfNotModified()
         $this->assertSame([null, []], $result);
     }
 
+    public function testLastModifiedIsEmptyFetch()
+    {
+        $this->__setUpReader($this->url);
+        $item = $this->_createItem();
+        $feed = $this->_createFeed($lastModified='0');
```

Comment:
php doesn't have named variables so this shouldn't work.

Reply:
Didn't know that anymore. Shouldn't this cause an error? Well I will change it with the next commit :grin: